core/domain/subtopic_page_domain.py

```diff
@@ -224,7 +276,10 @@ def create_default_subtopic_page(cls, subtopic_id, topic_id):
 
     @classmethod
     def convert_html_fields_in_subtopic_page_contents(
-            cls, subtopic_page_contents_dict, conversion_fn):
+            cls,
+            subtopic_page_contents_dict: SubtopicPageContentsDict,
+            conversion_fn: Callable[..., None]
+            ) -> SubtopicPageContentsDict:
```

Comment:
```suggestion
        cls,
        subtopic_page_contents_dict: SubtopicPageContentsDict,
        conversion_fn: Callable[..., None]
    ) -> SubtopicPageContentsDict:
```

Why `...`?

Reply:
I seemed the right choice. Could you please point me to the more suitable alternative?